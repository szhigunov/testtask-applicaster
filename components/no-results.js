import React from 'react';
import {View, Text, StyleSheet} from 'react-native';

export const NoResults = () => (
  <View style={styles.preloaderView}>
    <View>
      <Text style={styles.loadingMsg}>No Results Found</Text>
    </View>
  </View>
);

const styles = StyleSheet.create({
  preloaderView: {flex: 1, alignItems: 'center', justifyContent: 'center'},
  loadingMsg: {paddingTop: 10},
});
