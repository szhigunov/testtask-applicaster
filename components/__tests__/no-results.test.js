/**
 * @format
 */

import 'react-native';
import React from 'react';
import {NoResults} from '../no-results';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('NoResults renders correctly', () => {
  let root = renderer.create(<NoResults />);
  expect(root.toJSON()).toMatchSnapshot();
});
