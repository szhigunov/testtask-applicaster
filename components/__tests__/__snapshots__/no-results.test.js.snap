// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View>
    <Text
      style={
        Object {
          "paddingTop": 10,
        }
      }
    >
      No Results Found
    </Text>
  </View>
</View>
`;
